$NetBSD: patch-genmon_genmon_c,v 1.2 2016/03/13 09:06:01 dholland Exp $

- Use standard headers.
- Functions returning pointers must be declared before being called.
(The function in question does not appear to be in any header file
and it isn't clear which if any it should be added to.)

--- genmon/genmon.c.orig	1998-07-15 16:27:49.000000000 +0000
+++ genmon/genmon.c
@@ -55,6 +55,9 @@
 #include "nocol.h"
 #include <sys/file.h>			/* for access() defines */
 #include <string.h>
+#include <stdlib.h>
+
+EVENT *logstr_to_event(char *s);
 
 #ifndef SLEEPTIME
 # define SLEEPTIME  15			/* after each popen()  */
